Libs/ITKFactoryRegistration/CMakeLists.txt

```diff
@@ -68,7 +68,7 @@ configure_file(
 # Install headers
 # --------------------------------------------------------------------------
 if(NOT DEFINED ${PROJECT_NAME}_INSTALL_NO_DEVELOPMENT)
-  set(${PROJECT_NAME}_INSTALL_NO_DEVELOPMENT ON)
+  set(${PROJECT_NAME}_INSTALL_NO_DEVELOPMENT ${Slicer_INSTALL_NO_DEVELOPMENT})
```

Comment:
This should already be set in https://github.com/Slicer/Slicer/blob/59da0db16500d7fbf8b1288748e59c28fad6bab0/Libs/CMakeLists.txt#L94